core/domain/summary_services.py

```diff
@@ -743,3 +743,28 @@ def get_recently_published_exp_summary_dicts(limit):
         reverse=True)
 
     return get_displayable_exp_summary_dicts(summaries)
+
+
+def get_displayable_story_summary_dicts(story_summary):
```

Comment:
No, I meant declaring a method on the StorySummary domain object. Not a function in summary_services.

Reply:
Removed.